starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +15,12 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact()">Submit New Contact</button>
+<ul>
+  <li *ngFor='let contact of contacts'>
+    {{contact.name}}<br/>
+    {{contact.email}}<br/>
+    {{contact.phoneNumber}}<br/>
+    <img [src]=contact.image>
```

Comment:
You forgot the quotes: ```<img [src]="contact.image">```.
Furthermore, it is best practice to always include descriptive alt text (if the image is not purely decorative), e.g.: ```alt="Profile picture of {{ contact.name }}"```.